cmake/onnxruntime_providers.cmake

```diff
@@ -291,10 +291,10 @@ if (onnxruntime_USE_OPENVINO)
 
   # Below variables point to directories within the OpenVINO installation directory
   # whose value is set in INTEL_CVSDK_DIR variable by running the setupvars.sh script
-if (onnxruntime_USE_OPENVINO_BINARY)
-  if ($ENV{INTEL_CVSDK_DIR} MATCHES "2019.1")
+  if ($ENV{INTEL_CVSDK_DIR} MATCHES "2019.3")
```

Comment:
This check can be moved to Line 284 as 
```
if ( NOT $ENV{INTEL_CVSDK_DIR} MATCHES "2019.3" )
  message(FATAL_ERROR ...)
endif()
```
And the rest of the code no longer needs to be enclosed within if blocks for version checks